BP/common/scripted_modifiers/00_building_modifiers.txt

```diff
@@ -0,0 +1,25 @@
+﻿#Scripted Modifiers used for buildings
```

Comment:
Isn't this the exact same as in vanilla? Do we need to include it?

Reply:
It's not; I probably had to tweak something that got overtaken by events; I've listed a full code cleanup as one of my last tasks for #71, but it's probably best done after we have an alpha candidate. Fixed on this branch though.